### PR TITLE
Update dockerfiles

### PR DESCRIPTION
**Pull request by tobi812**

### CODE REVIEW

File: src/requirements.txt [hunk 1]
```diff
@@ -4,3 +4,4 @@ asgiref==3.2.7
 Django==3.0.7
 pytz==2020.1
 sqlparse==0.3.1
+gunicorn==20.0.4
```
Comment (ruimbribeiro) [inline src/requirements.txt:7]: Maybe we want to add `django-environ` here as we use it in the settings

**Review by ruimbribeiro — COMMENTED** [review]
I had to change the file:

`herbieapp/models/generated_models.py → src/herbieapp/models/generated_models.py`

```
from herbie_core.models import AbstractBusinessEntity
```
to 

```
from herbie_core.models.models import AbstractBusinessEntity
```
As we changed this in the Herbie core package; most probably a new generation of the models would fix this.

**Review by ruimbribeiro — COMMENTED** [review]
LGTM, just 2 small comments

File: src/requirements.txt (hunk 1 above)
Comment (tobi812) [reply src/requirements.txt:7]: :+1: Good point, will do that.

**Comment by tobi812** [comment]
> I had to change the file:
> 
> `herbieapp/models/generated_models.py → src/herbieapp/models/generated_models.py`
> 
> ```
> from herbie_core.models import AbstractBusinessEntity
> ```
> 
> to
> 
> ```
> from herbie_core.models.models import AbstractBusinessEntity
> ```
> 
> As we changed this in the Herbie core package; most probably a new generation of the models would fix this.

Aha, okay I will do that. Good catch :+1:

**Review by ruimbribeiro — APPROVED** [review]

**Pull request merged**